src/Rector/OcServerToOcpServerRector.php

```diff
@@ -62,7 +62,7 @@ public function refactor(Node $node): ?Node
     public function getRuleDefinition(): RuleDefinition
     {
         return new RuleDefinition(
-            'Change method calls from set* to change*.',
+            'Change method calls from OC::$server to OCP\Server::get.',
```

Comment:
There are more methods on the server class. This rector only cares about get and query.